pkg/console/subresource/configmap/trusted_ca.go

```diff
@@ -0,0 +1,32 @@
+package configmap
+
+import (
+	corev1 "k8s.io/api/core/v1"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+
+	"github.com/openshift/console-operator/pkg/api"
+	"github.com/openshift/console-operator/pkg/console/subresource/util"
+)
+
+const (
+	injectTrustedCABundleAnnotation = "config.openshift.io/inject-trusted-cabundle"
+)
+
+func DefaultTrustedCAConfigMap(cr *operatorv1.Console) *corev1.ConfigMap {
+	configMap := TrustedCAStub()
+	util.AddOwnerRef(configMap, util.OwnerRefFrom(cr))
+	return configMap
+}
+
+func TrustedCAStub() *corev1.ConfigMap {
+	meta := util.SharedMeta()
+	meta.Name = api.TrustedCAConfigMapName
+	meta.Annotations = map[string]string{
+		injectTrustedCABundleAnnotation: "true",
```

Comment:
Works like our other configmaps then, we make an empty one, but some controller somewhere is watching for this annotation & will populate us with the correct data.

Reply:
correct.. at least thats how it should be done

Reply:
> some controller somewhere is watching for this annotation & will populate us with the correct data

@benjaminapetersen the configmap injection controller is part of the cluster-network-operator.